src/store/modules/tabs/reducer.ts

```diff
@@ -0,0 +1,34 @@
+import { Reducer } from 'redux';
+
+import { TabState, Action } from './types';
+import initialState from './state';
+
+const reducer: Reducer<TabState> = (state = initialState, action: Action) => {
+  switch (action.type) {
+
+    case 'CLEAR_TABS':
+      return {
+        ...state,
+        activeTab: null,
```

Comment:
is it okay that we are using null? if we use null, we will constantly have to do a null check? is there anyway we can have a default empty value?

Reply:
maybe a tab with an empty string?

Reply:
I mean I agree, but I don't know what you could use as an initial value. This makes more sense to me because as long as the reducer does its job, if there are tabs there can never be a null active tab unless someone breaks typescript

Reply:
On my app your suggestion did not show up sorry